rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_accessor :date
+
+  belongs_to :book
+  belongs_to :person
+
```

Comment:
- [ ]  It will be great to delete these methods. This is because they are actually used in` active-record associations` in the rails framework. Since this is just pure ruby, they are not needed